Mathlib/Topology/Order/LowerUpperTopology.lean

```diff
@@ -20,20 +20,20 @@ right-closed left-infinite intervals.
 
 ## Main statements
 
-- `LowerTopology.t0Space` - the lower topology on a partial order is T₀
-- `LowerTopology.isTopologicalBasis` - the complements of the upper closures of finite
+- `IsLower.t0Space` - the lower topology on a partial order is T₀
+- `IsLower.isTopologicalBasis` - the complements of the upper closures of finite
   subsets form a basis for the lower topology
-- `LowerTopology.continuousInf` - the inf map is continuous with respect to the lower topology
+- `IsLower.continuousInf` - the inf map is continuous with respect to the lower topology
```

Comment:
```suggestion
- `Topology.IsLower.continuousInf` - the inf map is continuous with respect to the lower topology
```

Reply:
Again, I think I had that https://github.com/leanprover-community/mathlib4/commit/c3c74570a0dac68377b95f55e40a66a068e08a1a#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386L26 but @YaelDillies changed it to https://github.com/leanprover-community/mathlib4/commit/6ec78992b4f30a3441a210a37139c8409fe5fde7#diff-00624ef46078276636f22330988bc2becf8bef0223f4560eb2d00b4e19e54386R26